snmp/datadog_checks/snmp/data/default_profiles/aruba-switch.yaml

```diff
@@ -83,6 +84,26 @@ metrics:
           OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17.1.1
           name: sysExtFanIndex
 
+  - MIB: WLSX-SYSTEMEXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17
+      name: wlsxSysExtFanTable
+    symbols:
+      - name: wlsxSysExtFan
+        constant_value_one: true
+    metric_tags:
+      - tag: fan_index
+        column:
+          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17.1.1
+          name: sysExtFanIndex
+      - tag: fan_status
+        column:
+          OID: 1.3.6.1.4.1.14823.2.2.1.2.1.17.1.2
+          name: sysExtFanStatus
+        mapping:
+          1: active
+          2: inactive
+
```

Comment:
This seems like it's duplicating the previous block. Also I don't see `sysExtFanStatus` anywhere in the output when I `ddev env check` this.

Reply:
It's not a duplication but an intentional change. In the previous table we have a metric `sysExtFanStatus` with one tag, these were present here before. In the new table, I added a tag `fan_status` and collect the number of devices with specific tags (`fan_index` and `fan_status`), that's the purpose of `constant_value_one`. Cannot comment on `ddev env check`, but `sysExtFanStatus` is tested in e2e correctly,

Reply:
The `ddev env check` problem turned out to be a mistake on my end, it's looking fine now.